dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05200WorkflowTaskUniqueKey.java

```diff
@@ -36,6 +41,37 @@ public void executeUpgrade() throws DotDataException, DotRuntimeException {
             throw new DotDataException(e.getMessage(), e);
         }
         final DotConnect dotConnect = new DotConnect();
+        
+        List<Map<String,Object>> results = dotConnect.setSQL("select webasset,language_id from workflow_task group by webasset,language_id having count(*)>1").loadObjectResults();
+        
+        for(final Map<String,Object> map : results) {
+            final String webAsset = (String) map.get("webasset");
+            final long lang = Try.of(()->(Long) map.get("language_id")).getOrElse(0l);
+            final List<String> deleteMes = dotConnect
+            .setSQL("select id from workflow_task where webasset=? and language_id=? order by mod_date desc")
+            .addParam(webAsset)
+            .addParam(lang)
+            .setStartRow(1) //magic happens here
+            .loadObjectResults()
+            .stream()
+            .map(r->(String)r.get("id")).collect(Collectors.toList());
+            
+            String subquery = "'" + String.join("','" , deleteMes) + "'";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'subquery' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4666366)